examples/integration/integration_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	"io"
 	"io/ioutil"
 	"net/http"
+	neturl "net/url"
```

Comment:
This is a bit nickpicky, but please don't do this. If you want to name a variable `url`, consider just naming it `u` or `endpoint` or something.

Reply:
The previous case used a lot of url variables. Wouldn't it be confusing if didn't change the net/url, or rename these variables?

Reply:
Thanks for your review, the last commit I renamed the local url variable, so is it feasible?